support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/DigitalSubscriptionDirectPurchaseBuilder.scala

```diff
@@ -49,4 +51,22 @@ class DigitalSubscriptionDirectPurchaseBuilder(
 
   }
 
+  def overridePricingIfRequired(state: DigitalSubscriptionDirectPurchaseState) = {
+    state.product.amount
+      .map { amount =>
+        // TODO: Limit this specifically to people in the benefits test
```

Comment:
Would it be worth also checking it's over some minimum in order to avoid a malicious user constructing a payload with a very low price?